CHANGELOG.md

```diff
@@ -2,9 +2,16 @@
 
 ## 1.0.0-beta.8 (Unreleased)
 
+### New Features
+- Azure Communication Common Library
+ - Introduction of  a new struct `CommunicationIdentifierModel` to repesent a union type that is either a `communicationUser`, `phoneNumber`, or `microsoftTeamsUser`.  Along side this new object we introduced `CommunicationIdentifierSerializer` to serialize and deserialize valid JSON. 
+
 ### Breaking Changes
 - Azure Communication Common Library
  - Communication identifier `MicrosoftTeamsUserIdentifier` property `identifier` renamed to `userId` since identifier was too generic.
+ - Communication identifier `MicrosoftTeamsUserIdentifier` property `id` renamed to `rawId` to represent full MRI.
+ - Communication identifier `PhoneNumberIdentifier` property `id` renamed to `rawId` to represent full MRI.
+ - Removed `CallingApplicationIdentifier` as it is currently unused by any service, and format needs to be finalized. 
```

Comment:
If 'CallingApplicationIdentifier' is removed, why we need to do 'format needs to be finalized. '

Reply:
Also, that sounds more like something for internal discussion rather than customer-relevant info.

Reply:
Fair point I can remove ` and format needs to be finalized. `